Rules/UseConsistentIndentation.cs

```diff
@@ -181,6 +203,23 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
                         break;
 
                     case TokenKind.RParen:
+                        bool matchingLParenIncreasedIndentation = false;
+                        if (lParenSkippedIndentation.Count > 0)
+                        {
+                            matchingLParenIncreasedIndentation = lParenSkippedIndentation.Pop();
+                        }
```

Comment:
This would make it inconsistent with the existing style where there is one newline only between the many `case` statements, which makes it nice to read in logically separated blocks of reasonable size IMHO.

Reply:
Well the current PSSA codebase style is such that I won't argue. I think blocks should always be separated by newlines (and would have included another newline below the `if` below if I'd noticed), but not willing to spend too much energy on it

Reply:
I'm not a terrible fan of the PSSA style either TBH (especially the long methods where high level context is needed). I suggest a follow-up style PR to make the whole file consistent with this style or define something like an `editorconfig` file so that we could apply the preferred format across all files automatically.